boards/vs203/Kconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD
+    default "meshme" if BOARD_MESHME
```

Comment:
Why the default name of board is not VS203?

Reply:
copy pasta confusions xD

Reply:
fixed!